### PR TITLE
Feature/readd topic grid view content type

### PR DESCRIPTION
**Pull request by kentis**

### CODE REVIEW

File: h5p-bildetema-words-grid-view/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { H5P } from "./h5p/H5P.util";
+import { H5PWrapper } from "./h5p/H5PWrapper";
+import "./index.scss";
+
+(H5P as any).H5pBildetema = H5PWrapper;
```
Comment (StianHaugland1) [inline h5p-bildetema-words-grid-view/src/index.tsx:5]:
```suggestion
registerContentType("BildetemaWordsGridView", H5PWrapper);
```

File: h5p-bildetema-words-grid-view/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { H5P } from "./h5p/H5P.util";
```
Comment (StianHaugland1) [inline h5p-bildetema-words-grid-view/src/index.tsx:1]:
```suggestion
import { registerContentType } from "h5p-utils";
```

File: h5p-bildetema-words-grid-view/src/index.tsx (hunk 1 above)
Comment (StianHaugland1) [inline h5p-bildetema-words-grid-view/src/index.tsx:5]:
```suggestion
import { registerContentType } from "h5p-utils";
import { H5PWrapper } from "./h5p/H5PWrapper";
import "./index.scss";

registerContentType("BildetemaWordsGridView", H5PWrapper);
```

File: h5p-bildetema-words-grid-view/.storybook/main.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  addons: ["@storybook/addon-links", "@storybook/addon-essentials"],
+  core: {
+    builder: "webpack5",
```
Comment (StianHaugland1) [inline h5p-bildetema-words-grid-view/.storybook/main.js:7]: Should we use vite for building this one as well? `builder: "@storybook/builder-vite",`

Comment (kentis) [reply h5p-bildetema-words-grid-view/.storybook/main.js:7]: sure

**Pull request merged**